src/Command/CommandHelper.php

```diff
@@ -91,20 +91,27 @@ public static function begin(
 		bool $cleanupContainerCache = true,
 	): InceptionResult
 	{
-		if (!$allowXdebug) {
-			$xdebug = new XdebugHandler('phpstan');
-			$xdebug->setPersistent();
-			$xdebug->check();
-			unset($xdebug);
-		}
-
 		$stdOutput = new SymfonyOutput($output, new SymfonyStyle(new ErrorsConsoleStyle($input, $output)));
 
 		/** @var Output $errorOutput */
 		$errorOutput = (static function () use ($input, $output): Output {
 			$symfonyErrorOutput = $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output;
 			return new SymfonyOutput($symfonyErrorOutput, new SymfonyStyle(new ErrorsConsoleStyle($input, $symfonyErrorOutput)));
 		})();
+
+		if ($allowXdebug && !XdebugHandler::isXdebugActive()) {
+			$errorOutput->writeLineFormatted(sprintf('Note: You are running with "--xdebug" enabled, but the xdebug php-extension is not active.' . "\n" . '      The process will not halt at breakpoints.'));
```

Comment:
1) Please use `$errorOutput->getStyle()->note()` instead.
2) Correct capitalization of `Xdebug` please: https://xdebug.org/
3) `php-extension` should be `PHP extension`

Reply:
fixed thanks.

screenshot of the cli error for preview

![grafik](https://user-images.githubusercontent.com/120441/196914304-84ad1eec-c3d6-4c17-8edd-9fdf36fc1cb0.png)